src/ops/shell.rs

```diff
@@ -56,7 +56,7 @@ pub fn main(project: Project, opts: ShellOptions) -> OpResult {
     bash_cmd
         .args(&[
             "-c",
-            "exec \"$1\" internal__start_user_shell --shell-path=\"$2\" --shell-file=\"$3\"",
+            "exec \"$1\" internal start_user_shell --shell-path=\"$2\" --shell-file=\"$3\"",
```

Comment:
Before this change in this file, the PR should have failed CI. So either
- there is a test that should have failed, but didn't - we need to investigate and fix (possibly in this PR)
- there is no test that covers this - we should add one (not in this PR)

Reply:
Ok I checked about tests. First it seems the execution of this command wasn't really checked. I added some check about the command's state just here https://github.com/target/lorri/pull/353/files#diff-9cbb4bd47e53163015bb66adcbc9db2cR24

Then I checked if a test existed and covered these lines of code. But it didn't, in fact the test never covered this function. So I took the decision to execute a test with `assert_cmd` to launch it as a real user in his prompt in order to cover these lines of code.

Let me know what do you think about that ?